assets/javascripts/discourse/lib/chat-message-flag.js

```diff
@@ -0,0 +1,95 @@
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import I18n from "I18n";
+
+export default class ChatMessageFlag {
+  title() {
+    return "flagging.title";
+  }
+
+  customSubmitLabel() {
+    return "flagging.notify_action";
+  }
+
+  submitLabel() {
+    return "flagging.action";
```

Comment:
This should be something more specific to chat, like "Flag Message" instead of "Flag Post"

Reply:
Saw the other messages are all customized already, nice!